### PR TITLE
support absolute filenames

### PR DESCRIPTION
**Pull request by yyx990803**
solves #265

### CODE REVIEW

File: spec/BasicSpec.js [hunk 1]
```diff
@@ -571,6 +571,19 @@ describe('HtmlWebpackPlugin', function () {
...
 
+  it('allows you to use an absolute output filename', function (done) {
+    testHtmlPlugin({
+      entry: path.join(__dirname, 'fixtures/index.js'),
+      output: {
+        path: OUTPUT_DIR,
+        filename: 'index_bundle.js'
+      },
+      plugins: [new HtmlWebpackPlugin({
+        filename: path.resolve(OUTPUT_DIR, 'test.html')
```
Comment (jantimon) [inline spec/BasicSpec.js]: Would this also work for `path.resolve(OUTPUT_DIR, 'subfolder', 'test.html')`?

**Comment by jantimon** [comment]
Looks very promissing - I'll look into it.

File: spec/BasicSpec.js (hunk 1 above)
Comment (yyx990803) [inline spec/BasicSpec.js]: yes, it should work with arbitrary absolute paths.

**Comment by jantimon** [comment]
Had to fix one thing but looks good:
https://github.com/ampedandwired/html-webpack-plugin/tree/yyx990803-master

**Comment by jantimon** [comment]
I am on the road right now and my internet is quite bad - Probably won't be able to publish this before tonight.

**Pull request merged**